practice/src/css/components/card.css

```diff
@@ -60,3 +62,32 @@
   line-height: var(--line-height-lg);
   color: var(--secondary-text-color);
 }
+
```

Comment:
.card-add-product {
  padding-top: 111px;
  padding-bottom: 110px;
  
  Card add product have padding too large, you should use flex to center content for it